src/app/client/src/app/modules/groups/components/create-group/create-group.component.html

```diff
@@ -71,3 +71,5 @@ <h4 class="sbcard__title sb__ellipsis sb__ellipsis--two">{{sbcard.title}}</h4>
   </div>
 
 </div>
+
+<app-group-form *ngIf="showBoardSelectnModal" ></app-group-form>
```

Comment:
Rename **showBoardSelectnModal** to **showGroupCreateForm**

Reply:
Resolved